src/rasterstats/utils.py

```diff
@@ -146,3 +189,58 @@ def boxify_points(geom, rast):
         geoms.append(box(*window_bounds(win, rast.affine)).buffer(buff))
 
     return MultiPolygon(geoms)
+
+
+def get_latitude_scale(lat):
+    """get ratio of longitudal measurement at a latitiude relative to equator
+
+    at the equator, the distance between 0 and 0.008993216 degrees
+    longitude is very nearly 1km. this allows the distance returned
+    by the calc_haversine_distance function when using 0 and 0.008993216
+    degrees longitude, with constant latitude, to server as a scale
+    of the distance between lines of longitude at the given latitude
+
+    Args
+        lat (int, float): a latitude value
+    Returns
+        ratio (float): ratio of actual distance (km) between two lines of
+                       longitude at a given latitude and at the equator,
+                       when the distance between those lines at the equator
+                       is 1km
+    """
+    p1 = (0, lat)
+    p2 = (0.008993216, lat)
+    ratio = calc_haversine_distance(p1, p2)
+    return ratio
```

Comment:
We should remove the latitude correction concept from this PR. Let's keep the PRs focused on a single concept to avoid getting them all tangled up.

Reply:
Agreed, I think this got merged in because I was working out of my fork's master branch which has all the features from PRs I submitted here combined.